### PR TITLE
ES6 video player

### PR DESCRIPTION
**Pull request by SiAdcock**
## What does this change?

ES6ifies the video player bootstrap. Since this was a tiny conversion, and I was already in the area, I gave `video-container.js` a named export to fix the ESLint warning.

## What is the value of this and can you measure success?

Shiny

## Does this affect other platforms - Amp, Apps, etc?

No

## Tested in CODE?

No

### CODE REVIEW

File: tools/es5to6.json [hunk 1]
```diff
@@ -22,9 +22,7 @@
     "projects/common/modules/video/onward-container.js",
     "bootstraps/enhanced/gallery.js"
   ],
-  "Simon Adcock": [
-    "bootstraps/enhanced/media/video-player.js"
-  ],
+  "Simon Adcock": [],
```
Comment (gustavpursche) [inline tools/es5to6.json]: 👏

**Review by gu-stav — APPROVED** [review]
Looks good to me.

File: tools/es5to6.json (hunk 1 above)
Comment (SiAdcock) [inline tools/es5to6.json]: 🙇

File: static/src/javascripts/projects/commercial/modules/hosted/video.js [hunk 2]
```diff
@@ -186,6 +186,7 @@ export const initHostedVideo = (
                         require => {
                             resolve(
                                 require('bootstraps/enhanced/media/video-player')
+                                    .videojs
```
Comment (akash1810) [inline static/src/javascripts/projects/commercial/modules/hosted/video.js]: dear god prettier!

**Review by akash1810 — COMMENTED** [review]
💎

File: static/src/javascripts/projects/commercial/modules/hosted/video.js (hunk 2 above)
Comment (SiAdcock) [inline static/src/javascripts/projects/commercial/modules/hosted/video.js]: Just don't even

**Pull request merged**